app/controllers/concerns/foreman/controller/require_ssl.rb

```diff
@@ -12,6 +12,10 @@ module RequireSsl
       def require_ssl?
         SETTINGS[:require_ssl]
       end
+
+      def unattended_ssl?
+        URI.parse(Setting[:unattended_url]).scheme == 'https'
```

Comment:
Could we add
```suggestion
        SETTINGS[:require_ssl] &&
          URI.parse(Setting[:unattended_url]).scheme == 'https'
```
it would make sense and make the tests pass as they are getting running on `http`, but having `https` unattended url.

Reply:
Sure.